packages/api/src/models/webhook.ts

```diff
@@ -28,6 +33,22 @@ const WebhookSchema = new Schema<IWebhook>(
       required: true,
     },
     url: {
+      type: String,
+      required: false, // TODO: should this not be required?
```

Comment:
@wrn14897 this was existing, but I feel like url should have always been required no? It is in the interfaces and all submits etc - is there a reason not to?

Reply:
Its not required in EE. In OSS, the url should be required. But  I think we can just update the `IWebhook` for now

Reply:
sgtm 👍